packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -19,27 +22,22 @@ const tabTransition = '.16s ease-in-out';
 const tabStyles = css({
   display: 'grid',
   gridTemplateColumns: 'min-content 1fr min-content',
+  '&:hover, &:focus-visible, &:focus-within:not(:focus)': {
+    gridTemplateColumns: 'min-content 1fr min-content',
+  },
   alignItems: 'center',
-  paddingLeft: 12,
-  paddingRight: spacing[1],
-  gap: spacing[2],
+  gap: spacing[200],
 
-  // same as the border at the top
-  paddingBottom: '4px',
+  borderTop: `${spacing[100]}px solid var(--workspace-tab-top-border-color)`,
+  paddingBottom: spacing[100], // same as the top border
 
-  maxWidth: spacing[6] * 4,
-  minWidth: spacing[6] * 2,
-  height: 36,
+  maxWidth: spacing[800] * 6, // 192px
+  minWidth: spacing[1600] * 2, // 128px
```

Comment:
I think the only thing I'd change is make this 64px rather than 128px. I just so happen to have a lot of really short collection names and I find that leaves me with a lot of tabs with extra wasted space on the right, quickly filling up the tab bar unnecessarily.

Reply:
In my experience collections are often short nouns like orders, items, tickets, users, accounts, customers, products, clients and so on.